frontend/packages/console-shared/src/components/dynamic-form/fields.tsx

```diff
@@ -337,7 +337,7 @@ export const DropdownField: React.FC<FieldProps> = ({
       key={idSchema.$id}
       title={t('console-shared~Select {{title}}', { title: title || schema?.title || name })}
       selectedKey={formData}
-      items={items ?? {}}
+      items={(items as object) ?? {}}
```

Comment:
Because items comes from `const { items, title } = getUiOptions(uiSchema);` which returns items as string | string[] | array | object .... In this case this must be an object, this is was Dropdown requested before.

Reply:
Moved this cast to the line where getUiOptions is assigned to items